AK/Types.h

```diff
@@ -158,6 +158,13 @@ constexpr u64 TiB = KiB * KiB * KiB * KiB;
 constexpr u64 PiB = KiB * KiB * KiB * KiB * KiB;
 constexpr u64 EiB = KiB * KiB * KiB * KiB * KiB * KiB;
 
+constexpr u64 KB = 1000;
+constexpr u64 MB = KB * KB;
+constexpr u64 GB = KB * KB * KB;
+constexpr u64 TB = KB * KB * KB * KB;
+constexpr u64 PB = KB * KB * KB * KB * KB;
+constexpr u64 EB = KB * KB * KB * KB * KB * KB;
```

Comment:
Side note, but I've long tried to find a proper chance to rewrite this list (or rather, the one above) into the following format (simply because it makes more sense, that's also how we specify sizes everywhere else):

```suggestion
constexpr u64 KB = 1000;
constexpr u64 MB = 1000 * KB;
constexpr u64 GB = 1000 * MB;
constexpr u64 TB = 1000 * GB;
constexpr u64 PB = 1000 * TB;
constexpr u64 EB = 1000 * PB;
```

Since we are touching this file now anyways, maybe that's a good opportunity to change the list above as well.